src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -445,6 +448,21 @@ protected function getEntitiesByModule()
         return $entitiesByModule;
     }
 
+    public function getModules(): array
```

Comment:
I'm not 100% convinced about the method name since it doesn't return just standard modules, but that's probably ok as long as we change the visibility to `private` and add a docblock describing that it includes themes in the list as well.

Reply:
Thank you very much. I have fixed that in the latest commit.